python/pyarrow/tests/parquet/test_dataset.py

```diff
@@ -1557,20 +1557,10 @@ def is_pickleable(obj):
 
 @pytest.mark.pandas
 @parametrize_legacy_dataset
-def test_builtin_pickle_dataset(tempdir, datadir, use_legacy_dataset):
-    import pickle
+def test_pickle_dataset(tempdir, datadir, use_legacy_dataset, pickle_module):
     dataset = _make_dataset_for_pickling(tempdir, use_legacy_dataset)
     _assert_dataset_is_picklable(
```

Comment:
Can we inline `_assert_dataset_is_picklable()` here?

Reply:
Will update! Good idea.